jdi-light-vuetify-tests/src/test/java/io/github/epam/vuetify/tests/complex/ChipGroupsTests.java

```diff
@@ -82,13 +95,22 @@ public void slideChipGroupTest() {
 
     @Test
     public void multipleChipGroupTests() {
-        multipleChipGroup.show();
-
         List<String> valuesToTest = Arrays.asList(EXPECTED_CHIP_TEXTS.get(0), EXPECTED_CHIP_TEXTS.get(3),
                 EXPECTED_CHIP_TEXTS.get(4));
+        multipleChipGroup.show();
         multipleChipGroup.select(valuesToTest);
         multipleChipGroup.is().selected(valuesToTest);
         multipleChipGroup.deselect(valuesToTest);
         multipleChipGroup.is().deselected(valuesToTest);
     }
+
+    @Test
+    public void mandatoryChipGroupTest() {
```

Comment:
To show how work with select()/deselect() methods will be enough to check with one value and with several values.
This test is excessive.